### DISCUSSION TITLE
how to get trimesh from ShapeNet data

### DISCUSSION THREAD

**Discussion by weiminw** [pyg-team/pytorch_geometric#5430] [Q&A]
import torch
import torch_geometric
from torch_geometric.datasets import TUDataset, ShapeNet
import trimesh
import torch_geometric.transforms as T
dataset = ShapeNet(root="/Users/wangweimin/Workspaces/pytorch-geometric-datasets/SHAPENET", pre_transform=T.KNNGraph(k=2))

data = dataset[0]
--> mesh = torch_geometric.utils.to_trimesh(data)
#got error:```AttributeError: 'GlobalStorage' object has no attribute 'face'```

**Comment by rusty1s** [comment]
The `ShapeNet` dataset we provide in PyG is a point cloud dataset - it does not have any `face` information.
